cmd/version.go

```diff
@@ -0,0 +1,82 @@
+// Copyright © 2021 Kaleido, Inc.
```

Comment:
It's annoying, but the linter will want this to be this year (since this file was created/edited this year)